### PR TITLE
Adding post on creating rails multi-tenant site

### PR DESCRIPTION
**Pull request by nburkley**
New post on how we created a multisite setup with rails

### CODE REVIEW

**Comment by nburkley** [comment]
Can I get some eyes on this please - any feedback is appreciated.

ping @edenspiekermann/devs , @sigmarstern in particular.

**Comment by HugoGiraudel** [comment]
Wooow, your first article on the devblog. ✨

File: _posts/2016-09-09-rails-multitenant-site.md [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+  locale:   'de'
+  timezone: 'Europe/Berlin'
+otherdomain:
+  name:     'Other Doman'
+  email:    'info@otherdomain.co.uk'
+  locale:   'en'
+  timezone: 'Europe/London'
+```
+
+We wrap the site settings with a simple ruby class set it using an application controller concern.
```
Comment (sigmarstern) [inline _posts/2016-09-09-rails-multitenant-site.md]: Should it be mentioned that this "simple ruby class" is named `Site`. Would make it easier to understand the following snippet.

Comment (nburkley) [inline _posts/2016-09-09-rails-multitenant-site.md]: yup, good point. will do

**Pull request merged**